index.html

```diff
@@ -1560,24 +1558,24 @@ <h3 id="applying">
       </h2>
       <pre class="idl">
           dictionary WebAppManifest {
-             TextDirectionType? dir;
-             DOMString? lang;
-             USVString? name;
-             USVString? short_name;
-             USVString? description;
-             sequence&lt;ImageResource&gt;? icons;
-             sequence&lt;ImageResource&gt;? screenshots;
-             sequence&lt;USVString&gt;? categories;
-             DOMString? iarc_rating_id;
-             USVString? start_url;
-             DisplayModeType? display;
-             OrientationLockType? orientation;
-             USVString? theme_color;
-             USVString? background_color;
-             USVString? scope;
-             ServiceWorkerRegistration? serviceworker;
-             sequence&lt;ExternalApplicationResource&gt;? related_applications;
-             boolean? prefer_related_applications;
+             TextDirectionType dir;
+             DOMString lang;
+             USVString name;
```

Comment:
Thought bubble: This should possibly be `required` (though let's not change it right now because the existing algorithm doesn't require it, and I want to change the semantics as little as possible).

Reply:
Ok, let me file an issue

Reply:
https://github.com/w3c/manifest/issues/622